README.md

```diff
@@ -1 +1,67 @@
-# Zenith-Bootcamp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
- [ ] The README should contains an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.